pkg/compiler/gcc-compiler.go

```diff
@@ -65,11 +65,15 @@ var (
 
 	stringArgs = map[string]struct{}{
 		"-D":                     struct{}{},
+		"-Q":                     struct{}{},
+		"-L":                     struct{}{},
+		"-I":                     struct{}{},
```

Comment:
https://github.com/endocode/qmstr/blob/036ec45960df371522c7e86c7e4a627ca7d14ef7/pkg/compiler/gcc-compiler.go#L252

Reply:
Same for -L